src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -439,6 +444,23 @@ public function setCheckLoginOnServer($bool)
         $this->checkLoginOnServer = $bool;
     }
 
+    public function setCaseSensitiveEncoding($encoding)
```

Comment:
please add a docblock explaining that this is to control what collate should be used when querying for nodes. and mention that its only relevant for mysql and usually the autodetection of appending _bin to the charset is good enough.